.github/chronographer.yml

```diff
@@ -1,30 +1,10 @@
----
-
+branch-protection-check-name: Changelog entry
 action-hints:
-  # check-title-prefix: chng  # default: `{{ branch-protection-check-name }}: `
-  external-docs-url: https://pip.pypa.io/how-to-changelog
+  check-title-prefix: "Chronographer: "
+  external-docs-url: https://pip.pypa.io/dev/news-entry-failure
   inline-markdown: >
-    Check out https://pip.pypa.io/how-to-changelog
-
-branch-protection-check-name: Timeline protection
-
+    See https://pip.pypa.io/dev/news-entry-failure for details.
```

Comment:
@pradyunsg note that both URL and markdown options are rendered if specified, one after another. They are in the example config as a demonstration but you probably don't want to have both in a real-world install...

See https://github.com/pypa/pip/pull/11906/checks?check_run_id=12306117942 for now they are displayed right now.